libs.versions.toml

```diff
@@ -0,0 +1,161 @@
+hive3-service = { module = "org.apache.hive:hive-service", version.ref = "hive3Version" }
+httpcomponents-httpclient5 = "org.apache.httpcomponents.client5:httpclient5:5.2.1"
+immutables-value = "org.immutables:value:2.9.2"
+jackson-core = { module = "com.fasterxml.jackson.core:jackson-core", version.ref = "jacksonVersion" }
+jackson-databind = { module = "com.fasterxml.jackson.core:jackson-databind", version.ref = "jacksonVersion" }
+jackson211-core = { module = "com.fasterxml.jackson.core:jackson-core", version.ref = "jackson211Version" }
+jackson211-databind = { module = "com.fasterxml.jackson.core:jackson-databind", version.ref = "jackson211Version" }
+jackson211-module-paranamer = { module = "com.fasterxml.jackson.module:jackson-module-paranamer", version.ref = "jackson211Version" }
+jackson212-core = { module = "com.fasterxml.jackson.core:jackson-core", version.ref = "jackson212Version" }
+jackson212-databind = { module = "com.fasterxml.jackson.core:jackson-databind", version.ref = "jackson212Version" }
+jackson212-module-paranamer = { module = "com.fasterxml.jackson.module:jackson-module-paranamer", version.ref = "jackson212Version" }
+jackson213-core = { module = "com.fasterxml.jackson.core:jackson-core", version.ref = "jackson213Version" }
+jackson213-databind = "com.fasterxml.jackson.core:jackson-databind:2.13.4.2"
+jaxb-api = "javax.xml.bind:jaxb-api:2.3.1"
+jaxb-runtime = "org.glassfish.jaxb:jaxb-runtime:2.3.3"
+libraries-bom = "com.google.cloud:libraries-bom:24.1.0"
+microprofile-openapi-api = "org.eclipse.microprofile.openapi:microprofile-openapi-api:3.1"
+nessie-client = { module = "org.projectnessie.nessie:nessie-client", version.ref = "nessieVersion" }
+netty-buffer = "io.netty:netty-buffer:4.1.68.Final"
+object-client-bundle = "com.emc.ecs:object-client-bundle:3.3.2"
+orc-core = { module = "org.apache.orc:orc-core", version.ref = "orcVersion" }
+parquet-avro = { module = "org.apache.parquet:parquet-avro", version.ref = "parquetVersion" }
+parquet-column = { module = "org.apache.parquet:parquet-column", version.ref = "parquetVersion" }
+parquet-hadoop = { module = "org.apache.parquet:parquet-hadoop", version.ref = "parquetVersion" }
+pig = "org.apache.pig:pig:0.14.0"
+roaringbitmap = "org.roaringbitmap:RoaringBitmap:0.9.22"
+slf4j-api = "org.slf4j:slf4j-api:1.7.36"
+slf4j-simple = "org.slf4j:slf4j-simple:1.7.36"
+snowflake-jdbc = "net.snowflake:snowflake-jdbc:3.13.22"
+
+# test libraries
+assertj-core = "org.assertj:assertj-core:3.24.2"
+awaitility = "org.awaitility:awaitility:4.2.0"
+awssdk-iam = { module = "software.amazon.awssdk:iam", version.ref = "awsSdkVersion" }
+awssdk-s3control = { module = "software.amazon.awssdk:s3control", version.ref = "awsSdkVersion" }
+esotericsoftware-kryo = "com.esotericsoftware:kryo:4.0.2"
+flink115-connector-test-utils = { module = "org.apache.flink:flink-connector-test-utils", version.ref = "flink115Version" }
```

Comment:
Should probably prefer a version-range described above w/ a preferred version for all the Flink major versions as well.

Reply:
done